From: Jakub Steiner <jimmac@gmail.com>
Date: Fri, 1 Mar 2019 15:34:41 +0100
Subject: theme: improve legibility of error messages

- the login dialog uses warning_color, so be consistent
- nevertheless increase lightness of the error_color

Fixes issue https://gitlab.gnome.org/GNOME/gnome-shell/issues/1016

Origin: upstream, commit:36861edc897aac6d4b5dd8b354fc9feceb48888
Applied-Upstream: 3.32.1
---
 data/theme/gnome-shell-sass/_colors.scss | 2 +-
 data/theme/gnome-shell-sass/_common.scss | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/theme/gnome-shell-sass/_colors.scss b/data/theme/gnome-shell-sass/_colors.scss
index 447ad70..0a851f3 100644
--- a/data/theme/gnome-shell-sass/_colors.scss
+++ b/data/theme/gnome-shell-sass/_colors.scss
@@ -16,7 +16,7 @@ $link_visited_color: if($variant == 'light', darken($selected_bg_color, 20%), li
 $top_hilight: $borders_edge;
 
 $warning_color: #f57900;
-$error_color: #cc0000;
+$error_color: #ff8080;
 $success_color: if($variant == 'light', #33d17a, darken(#33d17a, 10%));
 $destructive_color: if($variant == 'light', #e01b24, darken(#e01b24, 10%));
 
diff --git a/data/theme/gnome-shell-sass/_common.scss b/data/theme/gnome-shell-sass/_common.scss
index 2ac87d4..f1e00e0 100644
--- a/data/theme/gnome-shell-sass/_common.scss
+++ b/data/theme/gnome-shell-sass/_common.scss
@@ -392,7 +392,7 @@ StScrollBar {
 
   .prompt-dialog-error-label {
     font-size: 10pt;
-    color: $error_color;
+    color: $warning_color;
     padding-bottom: 8px;
   }
 
